modules/mono/config.py

```diff
@@ -36,3 +36,13 @@ def get_doc_path():
 def is_enabled():
     # The module is disabled by default. Use module_mono_enabled=yes to enable it.
     return False
+
+
+def get_opts(platform):
+    from SCons.Variables import BoolVariable
+
+    return [
+        BoolVariable("generate_mono_glue", "Generate C# glue", False),
```

Comment:
Yes, I agree that we should have renamed it to `dotnet` from the start, keeping it as `mono` is all kinds of confusing!
In this case, though, the command line argument for the generator is `generate-mono-glue`, and this is meant to be an alias to that. Do we want to diverge this flag name? I don't have a super strong opinion either way, really.

Reply:
I've added further comments on this below - lookin at all the flags, yeah I agree it would be better to make them more consistent and dotnet-ish.

Reply:
I think we should rename the `generate-mono-glue` argument too. The plan is to rename the module with the _Editor unification_, at that time we can rename these options too. I was trying to anticipate the rename so we don't have to change the SCons arguments later, but I guess it may be better to be consistent with the current names.

Reply:
I see your point, yeah, it would be nice to have the right names now. We can add an alias to the `generate-mono-glue` argument in preparation for that, and deprecate it later (so it doesn't break everyone's scripts)

Reply:
I think we should do the mono -> dotnet rename before 4.2, and thus now-ish. I put it on the agenda for Thursday's meeting.